README.md

```
@@ -27,18 +28,28 @@ This project includes the boilerplate code for a GenLayer use case implementatio
+   bun dev
+   ```
+
+   **Using npm:**
+   ```shell
+   cd frontend
    npm install
    npm run dev
    ```
-   The terminal should display a link to access your frontend app (usually at http://localhost:5173/).
+
+   The terminal should display a link to access your frontend app (usually at http://localhost:3000/).
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)